documents/knowledge/elements.yaml

```diff
@@ -3752,6 +3752,62 @@
     str, num: zfill(a, b)
   vectorise: false
 
+- element: "∆ċ"
+  name: Nth Cardinal
+  arity: 1
+  description: Get the nth cardinal / convert number to words
+  overloads:
+    num: num_to_words(a)
+  vectorise: true
+  tests:
+    - "[0] : 'zero'"
```

Comment:
Rather than have all these test cases for simple numbers, I think we should a) smoosh them together into a single testcase on a list (because it vectorises) and b) remove a couple of these and use edgier cases instead. For example, what happens for 1103? -1103? 1.2?

Reply:
fun fact: copilot made the cases lol

Reply:
done.